implementations/micrometer-registry-statsd/src/main/java/io/micrometer/statsd/StatsdMeterRegistry.java

```diff
@@ -419,6 +419,7 @@ protected DistributionStatisticConfig defaultHistogramConfig() {
 
     /**
      * @deprecated queue size is no longer available since 1.4.0
+     * @return queue size
```

Comment:
While the original intent of the method was to return the queue size, it no longer does that, so it would be better for the JavaDoc to reflect that.
```suggestion
     * @return constant {@literal -1}
```

Reply:
@shakuzen Thanks for the feedback! I applied the suggested changes.